src/SFA.DAS.ApprenticeAccounts/DTOs/MyApprenticeship/MyApprenticeshipDto.cs

```diff
@@ -1,28 +1,16 @@
 ﻿using System;
 
 namespace SFA.DAS.ApprenticeAccounts.DTOs.MyApprenticeship;
-
-public class MyApprenticeshipsDto
+public class MyApprenticeshipDto
 {
-    public Guid Id { get; set; }
-    public long? Uln { get; set; }
-    public long? ApprenticeshipId { get; set; }
-    public string? EmployerName { get; set; }
-    public DateTime? StartDate { get; set; }
-    public DateTime? EndDate { get; set; }
-    public long? TrainingProviderId { get; set; }
-    public string? TrainingProviderName { get; set; }
-    public string? TrainingCode { get; set; }
-    public string? StandardUId { get; set; }
-    public DateTime CreatedOn { get; set; }
 
-    public static implicit operator MyApprenticeshipsDto(Data.Models.MyApprenticeship? source)
+    public static implicit operator MyApprenticeshipDto?(Data.Models.MyApprenticeship? source)
```

Comment:
Can the operator be after the properties please

Reply:
done